openshift_metrics/utils.py

```diff
@@ -223,14 +221,16 @@ def write_metrics_by_namespace(
     rows = []
     namespace_annotations = get_namespace_annotations()
     headers = [
-        "Namespace",
-        "Coldfront_PI Name",
-        "Report Start Date",
-        "Report End Date",
-        "_cpu_hours",
-        "_memory_hours",
-        "SU TYPE",
-        "SU Type Hours",
+        "Invoice Month",
+        "Project - Allocation",
```

Comment:
Is it worth adding the "Project - Allocation ID" now since it is just a copy of the Project - Allocation in OpenShift?

Reply:
Sure, I was going by the headers listed here https://github.com/nerc-project/operations/issues/199#issue-1863537443
but I can add allocation ID in this PR

Reply:
That was my mistake, I should not have added headers directly to the issue I should have linked to the live document where we are updating those things.